custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py

```diff
@@ -0,0 +1,50 @@
+import requests
+import datetime
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+from waste_collection_schedule.service.ICS import ICS
+
+import urllib
```

Comment:
obsolete?

Reply:
don't know - used erlangen_hoechstadt_de.py  as inspiration

Reply:
I didn't see that it is used, therefore should be obsolete. Please remove it and test it everythin is still working.